themes/admin/Sea_Green/css/sea_green-rtl.css

```
@@ -19367,17 +19320,17 @@ input[type=number] {
 #sidebar .subpanel.level-0 .menu-selected {
   background: linear-gradient(to left, #40ab46 0px, #40ab46 6px, #fff 6px, #fff 100%);
 }
+#sidebar .subpanel.level-0 .selected {
+  font-weight: 500;
+  background: linear-gradient(to left, #40ab46 0px, #40ab46 8px, #fff 8px, #fff 100%);
```

Stylelint (17.2.0)

[error] Unexpected unit (length-zero-no-unit)

(length-zero-no-unit)